x-pack/test/security_solution_cypress/config.ts

```diff
@@ -47,6 +47,7 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
         `--xpack.securitySolution.enableExperimental=${JSON.stringify([
           'chartEmbeddablesEnabled',
           'disableTimelineSaveTour',
+          'riskEnginePrivilegesRouteEnabled',
```

Comment:
NIT: If just a few tests need this configuration, is better to pass it directly to the test instead of setting it on the global ocnfiguration.

Reply:
@MadameSheema is that possible now? https://github.com/elastic/security-team/issues/7423 still looks to be open.

While those test-specific flags work for CI, they don't work locally, which is when the tests get written 😅 . Is the current workflow to write the test locally with this global flag, and then delete it before you push/merge? Or is there a better way?

Reply:
@rylnd  works locally if you execute the tests in headless mode instead of the visual one. I understand that for developing and testing purposes the visual mode is more convenient, but I'm afraid for now for that mode there is no easier way. The flag should be deleted before merging the code.

Reply:
I didn't know passing to the test was an option, thanks! 👍